lib/index.ts

```diff
@@ -135,11 +135,11 @@ export const createValidateEnvSchema = <
  */
 export const validateEnvSchema = <
   S extends BaseEnvSchema,
-  Customizations extends EnvSchemaCustomizations<S>
+  Customizations extends EnvSchemaCustomizations<S>,
 >(
   schema: S,
   container: Record<string, string | undefined> = process.env,
-  customize?: Customizations,
+  customize: Customizations | undefined = undefined,
```

Comment:
isn't the previous one exactly the same?

Reply:
It is, but since updating typescript it triggered an error where optional parameters can’t be after auto-assigned ones

The alternative here would be swapping parameters positions

Reply:
I don't think we need this rule.
In my opinion, at least, these are both the same, so we could disable this rule and use the `eslint` one.
```
    "default-param-last": "off",
    "@typescript-eslint/default-param-last": "error"
```
It supports optional parameters.
What do you think, @oliveirarleo , @lcsmuller ?

Reply:
Done, good suggestion